aws/aws.go

```diff
@@ -784,6 +784,10 @@ func (s *awsOps) Create(
 		return nil, cloudops.NewStorageError(cloudops.ErrVolInval,
 			"Invalid volume template given", "")
 	}
+	if vol.VolumeType == nil {
+		return nil, cloudops.NewStorageError(cloudops.ErrVolInval,
+			"Drive type not specified", "")
+	}
```

Comment:
nit: May be "Drive type not specified in the storage spec"?

Reply:
Done.